cloud/google/configs/ct-mirror-startcom.sh

```diff
@@ -0,0 +1,9 @@
+export INSTANCE_TYPE="mirror"
+export PROJECT="ct-mirror-startcom"
```

Comment:
Following the advice for naming our DNS frontends, I've been calling this log "startcom1".